homeassistant/helpers/entity_platform.py

```diff
@@ -226,6 +226,12 @@ async def async_setup(
         if not hasattr(platform, "async_setup_platform") and not hasattr(
             platform, "setup_platform"
         ):
+            # Bail out the setup for legacy notify services that implement
```

Comment:
This shouldn't be needed as we won't support YAML based entity platforms or legacy discovery based entity platforms. We'll only support config entry based entity platforms.

Reply:
removed it